bedrock/mozorg/templates/mozorg/home/includes/m24/media-springboard.html

```diff
@@ -26,12 +26,21 @@ <h3 class="m24-c-intro-title m24-t-md">{{ ftl('m24-home-headline-you-ai') }}</h3
       author='Laura Chambers',
       preview=ftl('m24-home-introducing-anonym'),
       image=resp_img(
-        url="img/home/2024/media/chambers.jpg",
+        url="img/home/2024/media/chambers-72.jpg",
+        srcset={
+          "img/home/2024/media/chambers-32.jpg": "32w",
+          "img/home/2024/media/chambers-64.jpg": "64w",
+          "img/home/2024/media/chambers-72.jpg": "72w",
```

Comment:
**Question/suggestion (non-blocking):** these are such small files that I do wonder if serving three different sizes is perhaps a bit too much of a micro optimization? What you have here works (so feel free to ignore me!)

I don't think we'd see much of a performance hit if we just served a 72px image for all view ports though 🙂